test/membrane_live_web/controllers/recordings_controller_test.exs

```diff
@@ -26,7 +26,7 @@ defmodule MembraneLiveWeb.RecordingsControllerTest do
 
     test "lists all recording", %{conn: conn, webinars: webinars} do
       conn = get(conn, Routes.recordings_path(conn, :index))
-      assert json_response(conn, 200)["webinars"] == webinars
+      assert Enum.sort(json_response(conn, 200)["webinars"]) == Enum.sort(webinars)
```

Comment:
This still doesn't resolve the issue of failed test check. Sometimes tests don't pass, at least locally.

Reply:
Right, I had to add sorting in the other test as well.